src/utils/common.ts

```diff
@@ -478,3 +478,13 @@ export function isCoin(assetContractId: AssetContractId): boolean {
       .coinContractId === assetContractId,
   );
 }
+/**
+* Clean options from members that cause issues when sending messages
+* or when signing transactions.
+*/
+export function getCleanOptions<T>(options: T): Omit<T, 'onCompiler' | 'onNode'> {
```

Comment:
I suggest to rename this function so it would describe what kind of options will be cleaned.